pkg/sessions/redis/redis_store.go

```diff
@@ -108,6 +108,7 @@ func buildSentinelClient(opts options.RedisStoreOptions) (Client, error) {
 		MasterName:       opts.SentinelMasterName,
 		SentinelAddrs:    addrs,
 		SentinelPassword: opts.SentinelPassword,
+		DB:               opts.SentinelDB,
```

Comment:
just so we have the some order. i would like to have the `DB` above the `SentinelPassword` like you did with the flag.

Reply:
I was struck by the fact that the option is called `DB` and not `SentinelDB` in the FailOverOptions. Therefore I looked into the redis documentation and realised this is a general option which might be interesting for all three redis clients (Standalone, Sentinel and Cluster). Therefore I would propose to introduce it to all three build functions. Furthermore, we do have testing for redis in place which should definitely be extended for this change. We use the `miniredis` and `minisentinel` packages to simulate the redis api in the unittests.

Reply:
Is there anything I can do about it?

Reply:
The DB is a parameter usually taken from the redis connection string (the last /{digit} is the DB number). This option is irrelevant for Redis Cluster as it only supports 1 Database anyways.

It's been working so far in the redis standalone because it has taken it from the connection string, since we can provide multiple sentinel URLs, I think it makes sense to not allow DB index in the sentinel URLs and have a DB parameter for it to prevent further confusion.